### DISCUSSION TITLE
Specific location for "!orphaned" and "!downloading" folders

### DISCUSSION THREAD

**Discussion by ADMECA** [Kalanyr/gogrepo#111] [Q&A] [answered]
Hi, my games are in the folder "/data/gog/games/" that is a NFS mount to a NAS. I would like that temporary folders "!orphaned" and "!downloading" are in "/tempo/gog/" in local hard drive

That is possible ?

**Comment by Kalanyr** [comment]
I'm happy to allow the redirection of !orphaned but ! downloading may have
significantly bad behaviour if it's not on the same drive as the actual
game followers ( the quick moves become long copies on big files ).

**Reply by ADMECA** [reply]
Ok, I understand, the problem is that the preallocation of large files (4GB) seems to be a problem on a network mount, and I don't have enough local disk space to hold my entire library. So for example when The Witcher 3 is updated, the preallocation of the 22x4GB files brings down my NFS. This is why I want to move the "!downloading" locally, because moving large files from local storage to NFS does not pose a problem, it is the preallocation that poses a problem. Is it possible to provide an option for ?

**Comment by Kalanyr** [comment]
I'll have to think about the design for this request, a change I made
recently might make this actually possible. I'm going to be away for about
a month on holiday so I'll get back to you in August.

**Reply by ADMECA** [reply]
Thank you for considering, and happy holidays :)

**Reply by ADMECA** [reply]
Hello, I'm coming to see the news :)

**Comment by Kalanyr** [comment, marked as answer]
I caught something on the plane back unfortunately so haven't had a solid
chance to look into this but I'll add it to the to-do.

**Reply by ADMECA** [reply]
No problem, many thanks.
